var/spack/repos/builtin/packages/py-tensorboard/package.py

```diff
@@ -84,7 +84,7 @@ class PyTensorboard(PythonPackage):
     depends_on("py-markdown@2.6.8:", type=("build", "run"))
     depends_on("py-numpy@1.12.0:", type=("build", "run"))
     depends_on("py-protobuf@3.6.0:3.19", type=("build", "run"), when="@:2.8")
-    depends_on("py-protobuf@3.9.2:3.19", type=("build", "run"), when="@2.9:")
+    depends_on("py-protobuf@3.9.2:3.20", type=("build", "run"), when="@2.9:")
```

Comment:
This is only true in TB 3.11, which hasn't been packaged yet. Before https://github.com/tensorflow/tensorboard/pull/5922 it was `<3.20`.